test/ServiceTest.cpp

```diff
@@ -168,77 +176,83 @@ TEST_F(ServiceTest, computeResult) {
   auto runComputeResult =
       [&](const std::string& result,
           boost::beast::http::status status = boost::beast::http::status::ok,
-          std::string contentType = "application/sparql-results+json")
-      -> std::shared_ptr<const Result> {
-    Service s{testQec, parsedServiceClause,
+          std::string contentType = "application/sparql-results+json",
+          bool silent = false) -> std::shared_ptr<const Result> {
+    Service s{testQec, silent ? parsedServiceClauseSilent : parsedServiceClause,
               getResultFunctionFactory(expectedUrl, expectedSparqlQuery, result,
                                        status, contentType)};
     return s.getResult();
   };
 
+  // Checks that a given result throws a specific error message, however when
+  // the `SILENT` keyword is set it will be caught.
+  auto expectThrowOrSilence =
+      [&](const std::string& result, std::string_view errorMsg,
+          boost::beast::http::status status = boost::beast::http::status::ok,
+          std::string contentType = "application/sparql-results+json") {
+        AD_EXPECT_THROW_WITH_MESSAGE(
+            runComputeResult(result, status, contentType),
```

Comment:
```suggestion
            runComputeResult(result, status, contentType, false),
```
Makes it much more readable.